src/test/scala/com/modelfabric/test/FusekiManager.scala

```diff
@@ -36,12 +36,14 @@ object FusekiManager {
     var process: Option[Process] = None
 
     override def run(): Unit = {
+      // JC: another option is to call org.apache.jena.fuseki.cmd.FusekiCmd.main(args), instead of starting a process
       val path = new org.apache.jena.fuseki.Fuseki().getClass.getProtectionDomain.getCodeSource.getLocation.getPath
       val cmd = s"java -jar $path --port=$port --mem --update $resource"
       println(s"Launching Fuseki Server: $cmd")
       process = Some(Runtime.getRuntime.exec(cmd))
     }
 
+    // JC: Java Thread class has start method. Why define a new method??
```

Comment:
the class name is FusekiRunner, I think start is clear enough :) anyway, not a big deal